src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,231 @@
+        Radius of the sky region around pointing position. Default: full sky
+    Vmag_cut: float
+        Return only stars above a given apparent magnitude. Default: None (all entries)
+    Bmag_cut: float
+        Return only stars above a given absolute magnitude. Default: None (all entries)
+
+    Returns
+    -------
+    Astropy table:
+       List of all stars after cuts with same keys as the input table stars
     """
-    Get an astropy table of bright stars.
 
-    This function is using the Yale bright star catalog, available through ctapipe
-    data downloads.
+    if Bmag_cut is not None and "Bmag" in stars.keys():
+        stars = stars[stars["Bmag"] < Bmag_cut]
+    elif Bmag_cut is not None and "BTmag" in stars.keys():
+        stars = stars[stars["BTmag"] < Bmag_cut]
+    if Vmag_cut is not None and "Vmag" in stars.keys():
+        stars = stars[stars["Vmag"] < Vmag_cut]
+    elif Vmag_cut is not None and "VTmag" in stars.keys():
+        stars = stars[stars["VTmag"] < Vmag_cut]
 
-    The included Yale bright star catalog contains all 9096 stars, excluding the
-    Nova objects present in the original catalog and is complete down to magnitude
-    ~6.5, while the faintest included star has mag=7.96. :cite:p:`bright-star-catalog`
+    if radius is not None:
+        if pointing is None:
+            raise ValueError(
+                "Sky pointing, pointing=SkyCoord(), must be "
+                "provided if radius is given."
+            )
+        separations = stars["ra_dec"].separation(pointing)
+        stars["separation"] = separations
+        stars = stars[separations < radius]
+
+    return stars
+
+
+def get_star_catalog(catalog, filename):
+    """
+    Utility function to download a star catalog for the get_bright_stars function
+
+    Parameters
+    ----------
+    catalog: string
+        Name of the catalog to be used. Usable names are found in the Enum StarCatalogues. Default: Yale
+    filename: string
+        Name and location of the catalog file to be produced
+    """
+    from astroquery.vizier import Vizier
+
+    vizier = Vizier(
+        catalog=StarCatalogues[catalog].value,
+        columns=[
+            "recno",
+            "RAJ2000",
+            "DEJ2000",
+            "RAICRS",
+            "DEICRS",
+            "pmRA",
+            "pmDE",
+            "Vmag",
+            "Bmag",
+            "BTmag",
+            "VTmag",
+        ],
+        row_limit=1000000,
+    )
+
+    stars = vizier.query_constraints(Vmag="0.0..10.0")[0]
+
+    if "Bmag" not in stars.keys():
+        log.exception("The chosen catalog does not have Bmag data")
+    if "Vmag" not in stars.keys():
+        log.exception("The chosen catalog does not have Vmag data")
+    stars.meta["Catalog"] = StarCatalogues[catalog].value
+
+    stars.write(CACHE_FILE, overwrite=True)
+
+
+def get_bright_stars(
+    pointing=None, radius=None, Bmag_cut=None, Vmag_cut=None, catalog="Yale"
+):  # max_magnitude):
```

Comment:
remove commented part

Reply:
For backwards compatibilty, we could think about keeping the `max_magnitude` argument and adding an `magnitude_type=` keyword that can either be `"Vmag"` or `"BMag"`